Sources/FaunaDB/Query.swift

```diff
@@ -26,6 +26,132 @@ public struct Ref: Fn {
 
 // MARK: Basic Forms
 
+/// Returns a native reference to classes. This allows for example,
+/// paginate over all classes in a database.
+///
+/// [Reference](https://fauna.com/documentation/queries#basic_forms).
+public struct Classes: Fn {
+
+    var call: Fn.Call
+
+    public init() {
+        self.call = fn("classes" => NullV())
+    }
+
+    /// - Parameter scope: the scope database.
+    public init(scope: Expr) {
```

Comment:
The most idiomatic form of optional in this driver would be:

```swift
public init(scope: Expr? = nil) {
    self.call = fn("classes" => scope)
}
```

Unfortunately, you can't use that form because `fn` discard `nil` to help us transparently represent optionals.

I think a better format would be a single constructor as the following:

```swift
public init(scope: Expr? = nil) {
    self.call = fn("classes" => (scope ?? NullV()))
}
```